src/components/ActivitiesListItem.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import {Link} from 'react-router-dom';
+
+const ActivitiesListItem = ({name, day, classNo, teacher, id}) => (
+  <div>
+    <Link to={`/edit/${id}`}>
+      <h3>{name}</h3>
+    </Link>
+    <p>On {day} on {classNo} class with {teacher}</p>
```

Comment:
`class` is important word. You can handle it by making this sentence as template string.